z2/src/deployer.rs

```diff
@@ -293,13 +300,16 @@ pub async fn run_deposit(config_file: &str, node_selection: bool) -> Result<()>
         let validator = validators::Validator::new(
             &node_ethereum_address.peer_id,
             &node_ethereum_address.bls_public_key,
-            &node_ethereum_address
-                .secret_key
-                .deposit_auth_signature(
-                    node.chain_id(),
-                    node_ethereum_address.secret_key.to_evm_address(),
-                )
-                .to_string(),
+            &hex::encode(
```

Comment:
Isn't it kind of silly to convert the signature to a string here, only for `Validator::new` to immediately parse it again to a signature?

Reply:
Yes, Im going to clean but wanted to get out the working version to unblock today

Reply:
yes, let's have a working version so that we can deposit and continue with the test on the devnet and you can refactor it later